database/src/main/java/com/firebase/ui/database/FirebaseRecyclerAdapter.java

```diff
@@ -57,7 +57,6 @@ public void startListening() {
     @OnLifecycleEvent(Lifecycle.Event.ON_STOP)
     public void stopListening() {
         mSnapshots.removeChangeEventListener(this);
-        notifyDataSetChanged();
```

Comment:
Is this change related to AGP7?

Reply:
@samtstern Kind of. Actually, lint got mad at this method call. It suggests calling a more specific one such as `notifyItemInserted`, `notifyItemDeleted`, `notifyItemMoved` etc. 

But I figured that since we're only removing the listener and not adding/removing any item, maybe there's nothing to notify the adapter about?

Reply:
SGTM